### DISCUSSION TITLE
plugin SimpleInventory is not registered

### DISCUSSION THREAD

**Discussion by ghost** [nornir-automation/nornir#841] [Q&A] [answered]
Hello,

i'm new with nornir and  trying to set it up after successfully using python/netmiko but i'm stuck at the begining.

Can someone help with this error i'm getting for simple .py script? 

script:
import os; print(os.getcwd()); print(os.listdir())
from nornir import InitNornir
nr = InitNornir(config_file="config.yml")
print(nr)

**Comment by ktbyers** [comment]
@IvanBajo Format in your config.yaml should be:

```yaml
---
inventory:
  plugin: SimpleInventory
```

**Comment by ktbyers** [comment]
What does your host file look like?

Can you cat this file:

"/home/ibajo/Desktop/nornir/inventory/hosts.yml"

You can obviously obscure any confidential information.

**Reply by ghost** [reply]
i can cat my hosts file from the path, and this is my hosts.yml

> ```yaml
>---
>test:
>    hostname: 1.1.1.1
>    groups: somegroup
> ```

**Reply by ktbyers** [reply, marked as answer]
The format of your groups section is wrong. Groups should be a YAML list.

```yaml
---
test:
   hostname: 1.1.1.1
   groups:
     - somegroup
 ```

It is possible you also have issues with your `groups.yaml` file, but try the above fix.

**Reply by ghost** [reply]
thank you very much

the groups.yml and/or default.yml had also something wrong so i copy/pasted all data from https://nornir.readthedocs.io/en/latest/tutorial/inventory.html#

and did print(nr.inventory.hosts) and get what i wanted:
{'host1.cmh': Host: host1.cmh, 'host2.cmh': Host: host2.cmh, 'spine00.cmh': Host: spine00.cmh, 'spine01.cmh': Host: spine01.cmh, 'leaf00.cmh': Host: leaf00.cmh, 'leaf01.cmh': Host: leaf01.cmh, 'host1.bma': Host: host1.bma, 'host2.bma': Host: host2.bma, 'spine00.bma': Host: spine00.bma, 'spine01.bma': Host: spine01.bma, 'leaf00.bma': Host: leaf00.bma, 'leaf01.bma': Host: leaf01.bma}
